Storage/Bookmarks/Bookmarks.swift

```diff
@@ -7,7 +7,7 @@ import Shared
 import Deferred
 import SwiftyJSON
 
-private let log = Logger.syncLogger
+private let log = Logger.storageLogger
```

Comment:
We made the choice to use a single logger for both because almost everything interesting about Sync problems actually happens in storage code, and the two intertwine. Did you make this choice for a reason?

Reply:
I didn't know that. It seems weird to have them combined in one log, but if you think its best, I can revert this.

Reply:
It might be clearer if it were called `syncAndStorageLog`, indeed :)